bluepymm/prepare_combos/parse_files.py

```diff
@@ -129,9 +129,9 @@ def read_mm_recipe_yaml(recipe_filename):
     for region in recipe['neurons']:
         for etype in region['traits']['etype'].keys():
             n_combos = len(mecombos)
-            mecombos.loc[n_combos, 'layer'] = region['traits']['layer']
-            mecombos.loc[n_combos, 'fullmtype'] = region['traits']['mtype']
-            mecombos.loc[n_combos, 'etype'] = etype
+            mecombos.loc[n_combos, 'layer'] = str(region['traits']['layer'])
+            mecombos.loc[n_combos, 'fullmtype'] = str(region['traits']['mtype'])
+            mecombos.loc[n_combos, 'etype'] = str(etype)
```

Comment:
It works but I wonder if there's a configuration to `yaml.safe_load(f)` to load the fields as string in the first place?

Reply:
I think @mgeplf created these yaml, so he may know how to do that better than me.

Reply:
They're usually created/edited by hand, afaik, so best to sanitize them like above, IMO.

Reply:
Ok, then it's good. Explicit is better than implicit.